node-server/src/database/__mocks__/dao.js

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable no-console */
-
 const testData = require('../../utils/test-data')
+const logger = require('../../../../common/utils/logger')
 
 class Dao {
   chats
   messages
 
   // eslint-disable-next-line no-unused-vars
   constructor(querier) {
-    console.log('Mock database created')
+    logger.test('Mock database created')
```

Comment:
This was intentionally changed to use console.log instead of the logger. This was done due to the fact that when running console.logs in the tests, the tests will then print the filename and linenum of the console.log() call.